ui/lib/kv/addon/components/page/secret/details.js

```diff
@@ -14,19 +14,50 @@ import { tracked } from '@glimmer/tracking';
  *  @secretPath={{this.model.path}}
  *  @secret={{this.model.secret}}
  *  @metadata={{this.model.metadata}}
+ *  @noMetadataPermission={{eq this.model.metadata 403}}
```

Comment:
see comment about about this being noReadMetadataPermissions if this is for queryRecord.